src/App.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import MainPage from './pages/MainPage';
+
+import { Film } from './api';
+
+type AppProps = {
+  films: Film[];
+};
+
+export default function App(props: AppProps) {
```

Comment:
Делай сразу деструктуризацию пропсов
```suggestion
export default function App({ films }: AppProps) {
```